<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportSelectionWindow</class>
 <widget class="QMainWindow" name="ReportSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор отчета</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnPersonnelReport">
      <property name="text">
       <string>Отчет по персоналу</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDepartmentReport">
      <property name="text">
       <string>Отчет по отделам</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPositionReport">
      <property name="text">
       <string>Отчет по должностям</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAssignmentReport">
      <property name="text">
       <string>Отчет по назначениям</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEducationReport">
      <property name="text">
       <string>Отчет по образованию</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnWorkExperienceReport">
      <property name="text">
       <string>Отчет по опыту работы</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
